index.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-restricted-globals */
+let listOfBooks = [];
+
+function addBook(title, author) {
+  listOfBooks = JSON.parse(localStorage.getItem('books')) || [];
+  listOfBooks.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(listOfBooks));
+}
+// eslint-disable-next-line no-unused-vars
+function removeBook(event) {
+  const remTitle = event.target.parentElement.firstElementChild.textContent;
+  listOfBooks = JSON.parse(localStorage.getItem('books'));
+  listOfBooks = listOfBooks.filter((book) => book.title !== remTitle);
+  localStorage.setItem('books', JSON.stringify(listOfBooks));
+  location.reload();
+}
```

Comment:
- [ ] For now if we have two book with same title then removing one of them cause removing other as well. Please fix that. (Hint: it's better to use `id` for each book and use `filter` on `id` rather than `title`).

### Created two book with same title

![Screenshot from 2021-07-06 20-38-41](https://user-images.githubusercontent.com/22434660/124624192-4d692b80-de9a-11eb-8090-1c2bf2e51c9d.png)

### After removing one

![Screenshot from 2021-07-06 20-39-23](https://user-images.githubusercontent.com/22434660/124624225-55c16680-de9a-11eb-83b8-6300cdc2fcd7.png)